keel-api/src/main/kotlin/com/netflix/spinnaker/keel/rest/ResourceController.kt

```diff
@@ -75,14 +61,13 @@ class ResourceController(
   }
 
   @PutMapping(
-    path = ["/{name}"],
     consumes = [APPLICATION_JSON_VALUE, APPLICATION_YAML_VALUE],
     produces = [APPLICATION_JSON_VALUE, APPLICATION_YAML_VALUE]
   )
   @PreAuthorize("@authorizationSupport.userCanModifySpec(#resource.metadata.serviceAccount)")
-  fun update(@PathVariable("name") name: ResourceName, @RequestBody resource: SubmittedResource<Any>): Resource<out Any> {
-    log.debug("Updating: $resource")
-    return resourcePersister.update(name, resource)
+  fun upsert(@RequestBody resource: SubmittedResource<Any>): Resource<out Any> {
+    log.debug("Upserting: $resource")
+    return resourcePersister.upsert(resource)
```

Comment:
Would it make sense to distinguish between a `200` and `201` response? Not really a big deal but it might help us debug things.

Reply:
How would I pass that back to the controller? I'm happy to do that.

Reply:
Yeah, that's a good question.

Reply:
I'll add a log statement for now.